swagger/v4/swagger.yaml

```diff
@@ -617,8 +617,6 @@ paths:
       responses:
         '200':
           description: successful
-        '422':
```

Comment:
I think this won't change if we add the test for unprocessable entity in spec/requests/swagger_docs/v4/irregular_incomes_spec.rb - if we do we would have to rerun the command to regenerate the swagger documentation